### DISCUSSION TITLE
在fly.io使用docker部署报错 "/.puppeteerrc.js" not found: not found

### DISCUSSION THREAD

**Discussion by RekiDunois** [DIYgod/RSSHub#12209] [Q&A] [answered]
### Routes

```routes
NOROUTE
```

### Full routes

```fullroutes
NOROUTE
```

### Related documentation

https://docs.rsshub.app/install/#bu-shu-dao-fly-io

### What is expected?

正常部署

### What is actually happening?

```bash
 => ERROR [chromium-downloader 3/5] COPY ./.puppeteerrc.js /app/                                         0.0s
------
 > [chromium-downloader 3/5] COPY ./.puppeteerrc.js /app/:
------
Error failed to fetch an image or build from source: error building: failed to compute cache key: "/.puppeteerrc.js" not found: not found
```

### Deployment information

Self-hosted

### Additional info

```shell
自己使用flyctl或者使用GitHub Action都会报此错误
```

### This is not a duplicated issue

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this bug has not already been reported

**Comment by TonyRL** [comment, marked as answer]
Make sure your fork contains this [.puppeteerrc.js](https://github.com/DIYgod/RSSHub/blob/master/.puppeteerrc.js) file while building docker image.
